### PR TITLE
Add port Dark Deity

### PR DESCRIPTION
**Pull request by JeodC**
# New Port for Dark Deity

## Game Information
- **Title**: Dark Deity
- **URL**: [Steam](https://store.steampowered.com/app/1374840/Dark_Deity/)

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] ROCKNIX

### Resolution Tests
Test all major resolutions:
- [x] Higher resolutions (e.g., 1280x720)

## Notes
Port is tagged `ultra`, relevant devices tested.

### CODE REVIEW

File: ports/darkdeity/Dark Deity.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Setup permissions
+$ESUDO chmod 666 /dev/tty1
```
Comment (Cebion) [inline ports/darkdeity/Dark Deity.sh:20]: Not used anymore
```suggestion
```

File: ports/darkdeity/Dark Deity.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+$ESUDO chmod 666 /dev/uinput
```
Comment (Cebion) [inline ports/darkdeity/Dark Deity.sh:21]: Not used anymore
```suggestion

```

File: ports/darkdeity/Dark Deity.sh [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+echo "Loading, please wait... (might take a while!)" > $CUR_TTY
```
Comment (Cebion) [inline ports/darkdeity/Dark Deity.sh:22]: Think about using pm_message in the future to include message output to cfw like muOS.

File: ports/darkdeity/Dark Deity.sh [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+# Display loading splash
+if [ -f "$GAMEDIR/patchlog.txt" ]; then
+    $ESUDO ./lib/splash "splash.png" 1 
+    $ESUDO ./lib/splash "splash.png" 2000
+fi
+
+# Run the game
+$GPTOKEYB "gmloader.aarch64" -c "./darkdeity.gptk" & 
+pm_platform_helper "gmloader.aarch64"
+./gmloader.aarch64 -c gmloader.json
```
Comment (Cebion) [inline ports/darkdeity/Dark Deity.sh:64]: As per talk in the crew channels there has been no consensus about the name change of gmloadernext.aarch64 to gmloader.aarch64.

So until all consent to the name change we will be using the one we've used for all previous ports.

**Review by Cebion — DISMISSED** [review]
License files missing:
See https://github.com/PortsMaster/PortMaster-New/tree/main/ports/tvruhh/tvruhh/license for gmloadernext licenses.

**Review by Cebion — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**